examples/uma-server/server.go

```diff
@@ -53,6 +53,14 @@ func main() {
 		vasp1.handleClientPaymentConfirm(c)
 	})
 
+	engine.POST("/api/uma/pay_invoice", func(c *gin.Context) {
+		vasp1.handlePayInvoice(c)
+	})
+
+	engine.POST("/api/uma/request_pay_invoice", func(c *gin.Context) {
```

Comment:
naming nit: Can we call this `request_invoice_payment` for clarity. I had @matthappens change his examples too.